### PR TITLE
Force uninstall of numpy to avoid error

### PR DESCRIPTION
**Pull request by Philmod**
`ImportError: Something is wrong with the numpy installation. While importing we detected an older version of numpy in ['/opt/conda/lib/python3.6/site-packages/numpy']. One method of fixing this is to repeatedly uninstall numpy until none is found, then reinstall this version.`

### CODE REVIEW

**Comment by Philmod** [comment]
Tesseract test is failing, looking...

File: Dockerfile [hunk 1]
```diff
@@ -135,6 +135,8 @@ RUN apt-get -y install zlib1g-dev liblcms2-dev libwebp-dev libgeos-dev && \
     pip install cartopy && \
     # MXNet
     pip install mxnet && \
+    # Uninstall any old numpy
+    pip uninstall numpy; \
```
Comment (rosbo) [inline Dockerfile]: for consistency, I would look replace the `;` by `&&`.

Comment (Philmod) [reply Dockerfile]: I was doing that in case `numpy` wasn't installed before, but I can change it if you prefer?

Comment (rosbo) [reply Dockerfile]: I think with the `;`, if any command before (like the pip install mxnet) fails, then the ones after can still succceed and the overall RUN command will succeed.

This means, we could have a successful build but w/o some packages installed.

**Comment by rosbo** [comment]
> One method of fixing this is to repeatedly uninstall numpy until none is found

Is there more than one version of numpy installed on the base image? Or is it one of the step in our build that installs a second one.

**Comment by Philmod** [comment]
The problem was fixed, probably by the dependencies picking up the new numpy version.

**Pull request closed without merging**